.github/workflows/benchmark_on_push.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           sudo apt install gfortran gcc libopenblas-dev
       - name: Install python dependencies
         run: |
-          python -m pip install --upgrade pip wheel setuptools virtualenv asv wget cmake casadi numpy
+          python -m pip install --upgrade pip wheel setuptools virtualenv asv==0.5.1 wget cmake casadi numpy
```

Comment:
Can you add a comment on why we pinned the version, and a link to the PR/issue in asv that should fix it?

Reply:
Yes sure, I'm adding one. Thanks 🙂